src/components/StatueList.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import StatueList from "./Statuelist";
+
+it("renders without crashing", () => {
+  const MockSlaverStatues = [
+    {
+      statueName: "Edward Colston",
+    },
+  ];
+
+  shallow(<StatueList statue={MockSlaverStatues} />);
+});
+
+it("renders name of statue", () => {
+  const MockSlaverStatues = [
+    {
+      statueName: "Edward Colston",
+    },
+  ];
+  const wrapper = shallow(<StatueList statue={MockSlaverStatues} />);
```

Comment:
Not sure `statue` is the right prop name. Got any alternatives?

Reply:
hmmm we were thinking 
- objectList
- StatueList (but we have used that name for a component) 
-StatueObjectList
-StatueInformation
-StatueData
-SlaverStatueData